packages/test-data/tests-unit/media-nested-type/media-nested-type.less

```
@@ -0,0 +1,51 @@
+  }
+}
+
+// Unknown media types are valid CSS non-matches, so they reorder too.
+@media (max-width: 500px) {
+  @media tester {
+    .g { color: red; }
+  }
+}
+
+// An escaped-string feature condition (Anonymous) must not be mistaken for a media type.
+@fc: ~"(max-width: 500px)";
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@fc`:" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)